assets/js/vue/components/information-product/InformationProductForm.vue

```diff
@@ -265,21 +284,27 @@ export default {
       submitBtnText: 'Submit',
       deleteConfirmationDialog: false,
       ipSuccessModalText: '',
+      productTypeOptions: null,
+      selectedProductTypeDescriptions: [],
```

Comment:
In edit mode, this adds two of the same descriptors in the list.. I did delete one, that was associated with this IP.

selectedProductTypes: [4, 4]
0: 4
1: 4